bootstrap/bootstrap/bootstrap.py

```diff
@@ -59,6 +60,54 @@ def read_config_file() -> Dict[str, Any]:
         }
         return config
 
+    def pull_core(self) -> None:
+        try:
+            screen = curses.initscr()
+            curses.noecho()
+            curses.cbreak()
+            curses_ui = True
+        except Exception:
+            curses_ui = False
```

Comment:
it's better to declare `curses_ui` as false in the function scope and pass in the exception, otherwise the scope of `curses_ui` is not clear.